### ISSUE TITLE
Dnyandeo: Headless CMS, 3 weeks

### ISSUE THREAD

**Issue by Dnyandeo33** [lab-brussels-1/home#405]
# headless cms
## Learning goals

- 🥚 Understanding what a content management system is, and why people use it 
- 🥚 Given an existing CMS installation fill in and publish a new piece of content
- 🥚 Perform a simple http call via postman to retrieve a piece of information.
- 🐣 Extend and adapt an existing CMS configuration
- 🐣 Perform a http call with filtering via Postman
- 🐣 Perform a http call with "include" via Postman
- 🐣 Perform a http POST call to create new data
- 🐣 Perform a http PUT call to update existing data
- 🐣 Configuring endpoint permissions (Authentication vs Authorization)
- 🐥 Design a data model from scratch based on user requirements

**Comment by Dnyandeo33** [comment]
##  Week 1
Sorry for the late check-in comment.

## I Need Help With:
What is the difference between API & web service?

## What went well?
 - I understand well headless CMS
 - Understood how to check API with path parameters and query parameters. 
 - Familiar with Postman API testing tool.
 - Understood well how to use Strapi 4

## Sunday Prep Work
Continue studying...

**Comment by danielhalasz** [comment]
> What is the difference between API & web service?

they are indeed very similar concepts, the main difference is that not all APIs are accessible over the web.
you can read more about it here: https://rapidapi.com/blog/api-vs-web-service/

**Comment by Dnyandeo33** [comment]
##  Week 1

## I Need Help With:
- Is it possible to create a Home page, Navbar, Buttons,  and other pages using strapi 4 ?
- What is GraphQL. 

## What went well?
- Everything about Strapi 4 - collation type went well. but I didn't try to display it. 
- Tried modifying data using API.

## Sunday Prep Work
Continue studying Single type and components.

**Comment by danielhalasz** [comment]
> * Is it possible to create a Home page, Navbar, Buttons,  and other pages using strapi 4 ?

no, strapi is just a tool to easily create an API and a database connected to it, that can be used by a frontend app to fetch data. you have to write the frontend separately, in any way you like

> * What is GraphQL.

it is another kind of query language, but we (and most of the web) are using the REST standard for APIs.

**Comment by danielhalasz** [comment]
@Dnyandeo33 how was week3 for the strapi module?

**Comment by Dnyandeo33** [comment]
@danielhalasz Sorry didn't write check-in because I was sick. 
Week3 
It was awesome, 
- I tried Single type and component type
- Using strapi I create some links, buttons and pages 
- Understood well how to use dynamic zone and internalization.
